app/models/thread.php

```diff
@@ -90,10 +114,79 @@ public function write(Comment $comment)
         
     }
 
+    public static function get($id)            
+    {
+        $db = DB::conn();
+
+        $row = $db->row('SELECT * FROM thread WHERE id = ?', array($id));
+
+        if(!$row) {
+            throw new RecordNotFoundException('no record found');
+        }
+
+        return new self($row);                    
+    }
+
     public static function countAll()
     {
         $db = DB::conn();
         return $db->value("SELECT COUNT(*) FROM thread");
-    }                          
+    }  
+
+
+    public function getComments()
+    {
+        $comments = array();
+        
+        $db = DB::conn();
+        $rows = $db->rows('SELECT * FROM comment WHERE thread_id = ? ORDER BY created ASC', array($this->id));
```

Comment:
You may remove ```ORDER BY created ASC' ```. The default sort order for MySQL is ascending, with smallest values first.